recipes/pytables/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version="3.2.2" %}
+
+package:
+  name: pytables
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/PyTables/PyTables.git
+  git_tag: v.{{ version }}
+  patches:
+    # remove march=native, fix library paths
+    - setup.patch          # [win]
+    # fix a syntax error
+    - blosc_macro.patch
+    # teach it to look for dlls in Library\bin on win
+    - tables_init.patch    # [win]
+    # remove march=native for more widely compatible builds
+    - march.patch          # [unix]
+
+build:
+  entry_points:
+    - pt2to3 = tables.scripts.pt2to3:main
+    - ptdump = tables.scripts.ptdump:main
+    - ptrepack = tables.scripts.ptrepack:main
+    - pttree = tables.scripts.pttree:main
+
+requirements:
+  build:
+    - python
+    - cython
+    - zlib                     # [win]
+    - lzo                      # [linux]
+    - bzip2
+    - numexpr
+    - hdf5
+    - setuptools
+    # building blosc is done internally.
+    # this is an old version, though.
+    # would be better to link against new version externally.
+    # See https://github.com/PyTables/PyTables/issues/451
+    # - blosc                  # [win]
+    - numpy
+  run:
+    - python
+    - argparse                 # [py26]
+    - numpy
+    - numexpr
+    - hdf5
+    - zlib
+    - setuptools
```

Comment:
Is this really a runtime dep too? You seem to have mapped the entrypoints nicely, so just wondering if you can remove setuptools here.